From 713fb639c8d4920ae7581908de8da9fbbe83f3ae Mon Sep 17 00:00:00 2001
From: Koen Kooi <koen@dominion.thruhere.net>
Date: Wed, 3 Apr 2013 08:27:09 +0200
Subject: [PATCH 04/14] beaglebone: HACK: raise USB current limit

Signed-off-by: Koen Kooi <koen@dominion.thruhere.net>
---
 include/configs/am335x_evm.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/configs/am335x_evm.h b/include/configs/am335x_evm.h
index 2ac89af..8de4cc9 100644
--- a/include/configs/am335x_evm.h
+++ b/include/configs/am335x_evm.h
@@ -149,6 +149,7 @@
 #endif
 
 #define CONFIG_BOOTCOMMAND \
+	"i2c mw 0x24 1 0x3e; " \
 	"run findfdt; " \
 	"mmc dev ${mmcdev}; if mmc rescan; then " \
 		"echo SD/MMC found on device ${mmcdev};" \
-- 
1.8.1.4

