From 8ef64f73b14e5e5657b740ecae8cf62db1511138 Mon Sep 17 00:00:00 2001
From: Koen Kooi <koen@dominion.thruhere.net>
Date: Wed, 3 Apr 2013 11:47:14 +0200
Subject: [PATCH 08/14] beaglebone: HACK: change mmc order to avoid u-boot
 crashing

Signed-off-by: Koen Kooi <koen@dominion.thruhere.net>
---
 include/configs/am335x_evm.h | 9 +++++----
 1 file changed, 5 insertions(+), 4 deletions(-)

diff --git a/include/configs/am335x_evm.h b/include/configs/am335x_evm.h
index 9051249..8dc3a8a 100644
--- a/include/configs/am335x_evm.h
+++ b/include/configs/am335x_evm.h
@@ -152,11 +152,12 @@
 #define CONFIG_BOOTCOMMAND \
 	"i2c mw 0x24 1 0x3e; " \
 	"run findfdt; " \
-	"if mmc dev 1; then " \
-		"echo eMMC found;" \
-		"setenv mmcdev 1;"\
+	"mmc dev 0; if mmc rescan ; then " \
+		"echo micro SD card found;" \
+		"setenv mmcdev 0;"\
 	"else " \
-		"echo No eMMC found;" \
+		"echo No micro SD card found, setting mmcdev to 1;" \
+		"setenv mmcdev 1;"\
 	"fi;" \
 	"mmc dev ${mmcdev}; if mmc rescan; then " \
 		"echo SD/MMC found on device ${mmcdev};" \
-- 
1.8.1.4

