From 59af9f65dd21d9a4e0c78478c5c0bd83e7d2da7e Mon Sep 17 00:00:00 2001
From: Koen Kooi <koen@dominion.thruhere.net>
Date: Wed, 3 Apr 2013 12:04:17 +0200
Subject: [PATCH 09/14] beaglebone: update bootpart variable after mmc scan

Signed-off-by: Koen Kooi <koen@dominion.thruhere.net>
---
 include/configs/am335x_evm.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/configs/am335x_evm.h b/include/configs/am335x_evm.h
index 8dc3a8a..489f882 100644
--- a/include/configs/am335x_evm.h
+++ b/include/configs/am335x_evm.h
@@ -159,6 +159,7 @@
 		"echo No micro SD card found, setting mmcdev to 1;" \
 		"setenv mmcdev 1;"\
 	"fi;" \
+	"setenv bootpart ${mmcdev}:2;" \
 	"mmc dev ${mmcdev}; if mmc rescan; then " \
 		"echo SD/MMC found on device ${mmcdev};" \
 		"if run loadbootenv; then " \
-- 
1.8.1.4

