From 88c09ce2919919d1943a85e9f9475d17b347702e Mon Sep 17 00:00:00 2001
From: Jason Kridner <jdk@ti.com>
Date: Thu, 4 Apr 2013 05:57:00 +0000
Subject: [PATCH 10/14] am335x_evm: enable gpio command

---
 include/configs/am335x_evm.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/configs/am335x_evm.h b/include/configs/am335x_evm.h
index 489f882..f2838ac 100644
--- a/include/configs/am335x_evm.h
+++ b/include/configs/am335x_evm.h
@@ -43,7 +43,7 @@
 
 /* commands to include */
 #include <config_cmd_default.h>
-
+#define CONFIG_CMD_GPIO
 #define CONFIG_CMD_ASKENV
 #define CONFIG_VERSION_VARIABLE
 
-- 
1.8.1.4

